File: Array/Remove Element/python/Remove_Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        slowIndex = 0
        for fastIndex in range(len(nums)):
            if nums[fastIndex] != val:
                nums[slowIndex] = nums[fastIndex]
                slowIndex += 1
        return slowIndex

File: Array/Remove Element/Java/Remove_Element.java
class Remove_Element {
    public int removeElement(int[] nums, int val) {
        int slowIndex = 0;
        for (int fastIndex=0; fastIndex<nums.length; fastIndex++) {
            if(nums[fastIndex] != val) {
                nums[slowIndex] = nums[fastIndex];
                slowIndex++;
            }
        }
        return slowIndex;
    }
}
